packages/trader/src/Modules/Trading/Containers/Multiplier/multiplier-options.tsx

```diff
@@ -52,7 +52,6 @@ const MultiplierOptions = observer(({ toggleModal }: TMultiplierOptions) => {
                 toggleModal={toggleModal}
             />
             <MultipliersInfo
-                // @ts-expect-error TODO: ts migration of <MultipliersInfo />
```

Comment:
I like the fact that you're coming back and resolve the TODOs.. good job @kate-deriv

Reply:
Thank you)